src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -143,9 +143,9 @@
     </DynamicEnumProperty.Metadata>
   </DynamicEnumProperty>
 
-  <DynamicEnumProperty Name="MinimumPlatformVersion"
-                DisplayName="Minimum OS version"
-                Description="Specifies the minimum runtime version of the selected operating system."
+  <DynamicEnumProperty Name="SupportedOSPlatformVersion"
```

Comment:
I can see how this is a potentially confusing property. Do we have docs for this? The best I could find are:

https://docs.microsoft.com/dotnet/standard/frameworks#os-version-in-tfms

Reply:
Agree, maybe @dsplaisted or @mhutch might know a docs to redirect users about its use.

Reply:
@terrajobst was also involved in this design.

We should have documentation for this, but I don't know if we do yet.